client/signup/steps/design-picker/let-us-choose.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled';
```

Comment:
I think this is the first place in the signup framework that uses `emotion`. I was a bit worried it'd include a whole bunch of extra bytes to the payload, but it doesn't look like it has. `emotion` must already be subtly included somewhere else, so I guess it's ok to start using it for our step styling 🤷

Reply:
Turns out it's already used a lot in the `woocommerce-install` flow: https://github.com/Automattic/wp-calypso/blob/79e0795af6e0a63504a61a0191ac700814cb4197/client/signup/steps/woocommerce-install/components/support-card/index.tsx#L10